### PR TITLE
Unsigned compose listener script

### PR DESCRIPTION
**Pull request by manasagowri**
Signed-off-by: mgowri <mgowri@redhat.com>

# Description
Release YAML - http://magna002.ceph.redhat.com/cephci-jenkins/latest-rhceph-container-info/RHCEPH-4.2.yaml
UMB Message - https://datagrepper.engineering.redhat.com/id?id=ID:ceph-downstream-jenkins-csb-storage-38251-1630584076680-1977:1:1:1:1&is_raw=true&size=extra-large

Please include Automation development guidelines. Source of Test case - New Feature/Regression Test/Close loop of customer BZs

# Checklist:

- [ ] Create a test case in Polarion reviewed and approved.
- [ ] Create a design/automation approach doc. Optional for tests with similar tests already automated.
- [ ] Review the automation design
- [ ] Implement the test script and perform test runs
- [ ] Submit PR for code review and approve
- [ ] Update Polarin Test with Automation script details and update automation fields
- [ ] If automation is part of Close loop, update BZ flag qe-test_coverage “+” and link Polarion test

### CODE REVIEW

File: pipeline/unsigned-compose-listener.groovy [hunk 1]
```diff
@@ -0,0 +1,117 @@
+/*
+    Pipeline script for storing the RC build information for RPM.
```
[the remark this answers is not in the document]
Comment (manasagowri) [reply pipeline/unsigned-compose-listener.groovy:2]: Done

File: pipeline/unsigned-compose-listener.groovy [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+                    reference: '',
+                    trackingSubmodules: false
+                ]],
+                submoduleCfg: [],
+                userRemoteConfigs: [[
+                    url: 'https://github.com/red-hat-storage/cephci.git'
+                ]]
+            ])
+            sharedLib = load("${env.WORKSPACE}/pipeline/vars/lib.groovy")
+            sharedLib.prepareNode()
```
[the remark this answers is not in the document]
Comment (manasagowri) [reply pipeline/unsigned-compose-listener.groovy:34]: Done.

File: pipeline/unsigned-compose-listener.groovy [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+                                "composes":
+                                [
+                                    "${platform}": composeUrl
+                                ]
+                            ]
+            releaseContent.put("latest", latestMap)
+            sharedLib.writeToReleaseFile(majorVer, minorVer, releaseContent)
+        }
+        else if(releaseContent.containsKey("latest")){
+            intCephVersion = Integer.parseInt(cephVersion.replaceAll("[^0-9]", ""))
```
[the remark this answers is not in the document]
Comment (manasagowri) [reply pipeline/unsigned-compose-listener.groovy:65]: Done.

**Review by psathyan — APPROVED** [review]
Looks good to me

**Pull request merged**